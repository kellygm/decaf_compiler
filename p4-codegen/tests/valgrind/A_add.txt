==3516375== Memcheck, a memory error detector
==3516375== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==3516375== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==3516375== Command: ../decaf inputs/add.decaf
==3516375== 
5RETURN VALUE = 5
==3516375== 
==3516375== HEAP SUMMARY:
==3516375==     in use at exit: 0 bytes in 0 blocks
==3516375==   total heap usage: 1,910 allocs, 1,910 frees, 420,012 bytes allocated
==3516375== 
==3516375== All heap blocks were freed -- no leaks are possible
==3516375== 
==3516375== For lists of detected and suppressed errors, rerun with: -s
==3516375== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
