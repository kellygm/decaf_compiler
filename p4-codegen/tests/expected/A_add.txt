5RETURN VALUE = 5
